File: src/test/java/com/letgo/book/unit/domain/BookTitleMother.java
package com.letgo.book.unit.domain;

import com.letgo.book.domain.BookTitle;

final public class BookTitleMother {
    public static BookTitle create(String title) {
        return BookTitle.create(title);
    }

    public static BookTitle random() {
        return BookTitleMother.create("random");
    }
}

File: src/test/java/com/letgo/book/unit/domain/BookTitleMother.kt
package com.letgo.book.unit.domain

import com.letgo.book.domain.BookTitle
import java.time.LocalDateTime
import java.util.UUID

object BookTitleMother {
    fun create(
        title: String = UUID.randomUUID().toString(),
        createdAt: LocalDateTime = LocalDateTime.now(),
    ): BookTitle {
        return BookTitle(title, createdAt)
    }

    fun random(): BookTitle {
        return create()
    }
}
